user_docs/en/userGuide.t2t

```diff
@@ -961,101 +969,77 @@ This option is only available for installed copies of NVDA.
-If there is an error loading the synthesizer, NVDA will notify you with a message, and continue using the previous synthesizer.
-
-==== Synthesizer ====[SynthesizerSelectionSynthesizer]
-This option allows you to choose the synthesizer you wish NVDA to use for speech output.
-
-For a list of the Synthesizers that NVDA supports, please see the [Supported Speech Synthesizers #SupportedSpeechSynths] section.
-
-One special item that will always appear in this list is "No speech", which allows you to use NVDA with no speech output whatsoever.
-This may be useful for someone who wishes to only use NVDA with braille, or perhaps to sighted developers who only wish to use the Speech Viewer.
-
-==== Output device ====[SynthesizerSelectionOutputDevice]
-This option allows you to choose the sound card that NVDA should instruct the selected synthesizer to speak through.
-
-%kc:setting
-==== Audio Ducking Mode ====[SynthesizerSelectionDuckingMode]
-Key: NVDA+shift+d
-
-On Windows 8 and above, this option allows you to choose if NVDA should lower the volume of other applications while NVDA is speaking, or all the time while NVDA is running.
-- No Ducking: NVDA will never lower the volume of other audio. 
-- Duck when outputting speech and sounds: NVDA will only lower the volume of other audio when NVDA is speaking or playing sounds. This may not work for all synthesizers. 
-- Always duck: NVDA will keep the volume of other audio lower the whole time NVDA is running.
--
-
-This option is only available if NVDA has been installed.
-It is not possible to support audio ducking for portable and temporary copies of NVDA.
-
-+++ Voice Settings (NVDA+control+v) +++[VoiceSettings]
-The Voice Settings dialog, found in the Preferences menu, contains options that let you change the sound of the speech.
++++ Speech Settings (NVDA+control+v) +++[SpeechSettings]
+The speech category in the NVDA Settings dialog contains options that let you change the speech synthesizer as well as sound of the speech.
 For a quicker alternative way of controlling speech parameters from anywhere, please see the [Synth Settings Ring #SynthSettingsRing] section.
 
-The Voice Settings dialog box contains the following options:
+The Speech Settings category contains the following options:
 
-==== Voice ====[VoiceSettingsVoice]
-The first option that you land on in this dialog is a combo box listing all the voices of the current synthesizer that you have installed.
+==== Change synthesizer ====[SpeechSettingsChange]
+The first option in the Speech Settings category is the Change... button. This button activates the [Select Synthesizer #SelectSynthesizer] dialog, which allows you to select the active speech synthesizer and output device.
+This dialog opens on top of the NVDA Settings dialog.
+Saving or dismissing the settings in the Select Synthesizer dialog will return you to the NVDA Settings dialog.
+
+==== Voice ====[SpeechSettingsVoice]

```

Comment:
I'll need to talk to @Qchristensen about these anchor tags, going forward it would be best if we could avoid changing them unless they really are pointing to quite unrelated content. In this case the name has changed, but the content is more or less the same. The reason for wanting to avoid changing this is so that training material is able to link to a static part of the user guide, and the links will continue to work for as long as possible. That said, there is no released training material that relies on this yet.

Reply:
Your last point is indeed why I took the freedom to change these anchors. These anchors are still quite new and recent.